### PR TITLE
Implement (mobile-responsive) navbar and homepage sections

### PR DESCRIPTION
**Pull request by ptwu**
### Summary
This PR implements a navbar that resizes to a hamburger button in small screen widths, a homepage hero section + description row with 3 icons, and a footer.

### Screenshots
Initial UI Dev screenshot

### Test Plan
Run `mvn package appengine:run` in the root directory to get a test server running on Cloud Shell (or open index.html with your browser). See that the navbar works and is mobile-responsive

### Notes
Diff count is large because I added navbars to the `my-trips.html` and `trips-network.html` pages, which are empty for now. Sorry about that

### CODE REVIEW

**Comment by JoshLew7** [comment]
Do we have a build ID that we can use or are those not ready yet?

File: src/main/webapp/index.html [hunk 1]
```diff
@@ -1,34 +1,114 @@
...
       rel="stylesheet"
       href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css"
     />
   </head>
   <body>
-    <div class="row">
-      <div class="col s12 m4">
-        <div class="card">
-          <div class="card-image">
-            <img src="assets/img/placeholder.png" />
```
Comment (mffabrigas) [inline src/main/webapp/index.html:21]: Add blank alt tag according to HTML style guide (alt="")

**Comment by ptwu** [comment]
> Do we have a build ID that we can use or are those not ready yet?

We should ask about this in the pod meeting, I think they should have one set up for us.

File: src/main/webapp/index.html (hunk 1 above)
Comment (ptwu) [reply src/main/webapp/index.html:21]: this is deleted code (was just a placeholder)

**Comment by JoshLew7** [comment]
> > Do we have a build ID that we can use or are those not ready yet?
> 
> We should ask about this in the pod meeting, I think they should have one set up for us.

Ok sounds good

File: src/main/webapp/index.html (hunk 1 above)
Comment (mffabrigas) [reply src/main/webapp/index.html:21]: Nvm just noticed this was deleted

File: src/main/webapp/index.html [hunk 2]
```diff
@@ -1,34 +1,114 @@
...
-            <span class="card-title">Testing out Materialize</span>
+    <nav class="blue darken-2" role="navigation">
+      <div class="nav-wrapper container">
+        <a href="#" class="brand-logo">Overnightly</a>
```
Comment (mffabrigas) [inline src/main/webapp/index.html:24]: What does # link mean

**Pull request merged**